cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/TimestampOption.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cassandra.spark.bulkwriter;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+public final class TimestampOption implements Serializable
+{
+    private static final TimestampOption NOW = new TimestampOption(System.nanoTime() / 1000);
```

Comment:
I do not think you should use it to get the absolute time `now`. See the comment of the method. 

> This method can only be used to measure elapsed time and is not related to any other notion of system or wall-clock time. 

It should use `System#currentTimeMillis()`, which is used in C* codebase to generate timestamp for cells too.

Reply:
We need to have timestamp in `microseconds` to emulate the Timestamp option in spark connector. I thought Sytem.nanoTime() /1000 would be a more accurate value than System.currentTimeMillis() * 1000.